src/enclave/interface.h

```diff
@@ -40,6 +40,9 @@ struct CCFConfig
   std::string domain;
   size_t snapshot_tx_interval;
 
+  // Only if joining or recovering
```

Comment:
nic: can you please expand on this comment.

Reply:
Unless I'm missing something, this is where we store the snapshot we start/restore from, hence only on joining or recovering. That's hopefully something we can reflect in the name, maybe 'startup_snapshot' or 'snapshot_to_initialise_from'...

Reply:
Good point. Renamed to `startup_snapshot`.